Assets/_Project/Scripts/GameManager.cs

```diff
@@ -28,6 +28,7 @@ public class GameManager : MonoBehaviour
         private void Awake()
         {
             Instance = this;
+            DontDestroyOnLoad(Instance);

```

Comment:
You need to handle destroy for other scenes that have the game manager.

https://gist.github.com/yolanother/3ae865b6a80f2f45940f18645c308f20

Reply:
done